tests/test_cards.py

```diff
@@ -0,0 +1,27 @@
+from copy import deepcopy
+
+import pytest
+
+import items
+from ansi_tags import ansiprint
+
+
+# Dynamically generate a list of all the cards.
+# Avoids interference with other tests by not using the global items.cards.
+def get_all_cards() -> list[items.Card]:
+    return [card() for card in items.Card.__subclasses__()]
+
+
+local_copy_of_cards = get_all_cards()
+
```

Comment:
You can put `@pytest.mark.only` here when running the tests locally to skip everything else

Reply:
Got it.